### PR TITLE
Release/1.7.0

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Release Changelog for the data model library
+
+## 1.7.0
+
+* Provides new classes that describe incoming Oxford Nanopore instrument data structures
+* Provides new classes that describe a Oxford Nanopore Experiment(`OxfordNanoporeExperiment.class`) and its containing Oxford Nanopre Measurements (OxfordNanoporeMeasurement.class)
+* Provides a new method in the `SampleCodeFunction.class` `public static List<String> findAllQbicSampleCodes(String text)` that can be used to find all QBiC sample identifiers in a String object
```
Comment (KochTobi) [inline CHANGELOG.md:7]:
```suggestion
* Provide new classes that describe incoming Oxford Nanopore instrument data structures
* Provide new classes that describe a Oxford Nanopore Experiment(`OxfordNanoporeExperiment.class`) and its containing Oxford Nanopre Measurements (OxfordNanoporeMeasurement.class)
* Provide a new method in the `SampleCodeFunction.class` `public static List<String> findAllQbicSampleCodes(String text)` that can be used to find all QBiC sample identifiers in a String object 
```

**Review by KochTobi — DISMISSED** [review]
Changelog wording. Otherwise looks good

**Review by KochTobi — APPROVED** [review]
Looks good now.

**Comment by sven1103** [comment]
🎉

**Comment by KochTobi** [comment]
warum steht da `merged commit 996b3a3 into master` und nicht `merged release/1.7.0` into master?

**Pull request merged**